client/src/components/map/controls/settings/index.tsx

```diff
@@ -19,47 +18,36 @@ export const SettingsControl: FC<SettingsControlProps> = ({
   className,
   children,
 }: SettingsControlProps) => {
+  const [open, setOpen] = useState(false);
+
   return (
     <div className={cn('flex flex-col space-y-0.5', className)}>
-      <Popover>
-        <Tooltip>
-          <PopoverTrigger asChild>
-            <TooltipTrigger asChild>
-              <button
-                className={cn({
-                  [CONTROL_BUTTON_STYLES.default]: true,
-                  [CONTROL_BUTTON_STYLES.hover]: true,
-                  [CONTROL_BUTTON_STYLES.active]: true,
-                })}
-                aria-label="Map style"
-                type="button"
-              >
-                <Settings className="h-[20px] w-[20px]" />
-              </button>
-            </TooltipTrigger>
-          </PopoverTrigger>
-
-          <TooltipPortal>
-            <TooltipContent side="left" align="center">
-              <div className="font-serif text-2xs">Map settings</div>
-              <TooltipArrow className="fill-white" width={10} height={5} />
-            </TooltipContent>
-          </TooltipPortal>
-
-          <PopoverContent side="left" align="start" className="w-[240px] p-6" sideOffset={16}>
-            {children}
-            <PopoverClose asChild>
-              <Button
-                type="button"
-                size="icon-sm"
-                className="absolute right-3 top-3 rounded-lg bg-gray-50"
-              >
-                <span className="sr-only">Close</span>
-                <X className="h-4 w-4" />
-              </Button>
-            </PopoverClose>
-          </PopoverContent>
-        </Tooltip>
+      <Popover open={open} onOpenChange={setOpen}>
+        <PopoverTrigger asChild>
+          <button
+            className={cn({
+              [CONTROL_BUTTON_STYLES.default]: true,
+              [CONTROL_BUTTON_STYLES.hover]: true,
+              [CONTROL_BUTTON_STYLES.active]: open,
+              [CONTROL_BUTTON_STYLES.focus]: true,
+              [CONTROL_BUTTON_STYLES.disabled]: true,
+            })}
+            aria-label="Map settings"
+            type="button"
+          >
+            <Settings className="h-5 w-5" />
+          </button>
+        </PopoverTrigger>
+
+        <PopoverContent side="left" align="start" className="w-[240px] p-6" sideOffset={16}>
```

Comment:
We removed the Tooltip here. I guess is not needed anymore

Reply:
The `PopoverContent` component displays the Map settings. We're still showing this 🙂.

Reply:
Yes, I was talking about the Tooltip as in staging: 
<img width="239" alt="image" src="https://github.com/Orcasa-Platform/orcasa/assets/9701591/b5ef546a-7ced-44e3-a473-6ab749d699d8">

Reply:
Yes, I removed it because it was not part of the design anymore and I don't remember any specific request from CIRAD.